src/routes/docs/products/sites/quick-start/+page.markdoc

```diff
@@ -0,0 +1,86 @@
+---
+layout: article
+title: Start with Sites
+description: Get started quickly with Appwrite Sites. Follow a step-by-step guide to create your first Appwrite Site and deploy a web app.
```

Comment:
This page is broken, I can't access the preview. 

I would also include the some of the quick start cards for the top frameworks in the overview page. I would then add a link below those to this page to explore more framework quick starts. This will be great for product discoverability.

Reply:
Shall I add those for Next, Nuxt, SvelteKit, and React?